examples/stable-diffusion/image_to_image_generation.py

```diff
@@ -290,7 +291,7 @@ def main():
         kwargs["torch_dtype"] = torch.bfloat16
 
     if args.throughput_warmup_steps is not None:
-        kwargs["throughput_warmup_steps"] = args.throughput_warmup_steps
+        res["throughput_warmup_steps"] = args.throughput_warmup_steps
```

Comment:
There are both `kwargs` and `res` in this example, that's strange. I guess `res` originally stands for "resolution".
I think we should only have one no? And and let's keep it called `kwargs`, it's more explicit.